domains/event-store/poc/EventStore/IEventStore.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using EventStore.Serialization;
+
+namespace EventStore;
+
+public interface IEventStore<T> where T : EventModel {
+    void Produce(T model, IEnumerable<string> topics);
+
+    IEventConsumer<T> MakeConsumer(string topicPrefix);
+    IEventConsumer<T> MakeConsumer(string topicPrefix, DateTime fromDate);
```

Comment:
Could we use DateTimeOffset in stead of DateTime here? DateTimeOffset will always be a single point in time.

Reply:
change to pointer